source/getting_started/aiot_intelligent_edge/coco_for_applications/end_user_client_applications/android_end_user_client_applications/quick_start_guide_android_end_user_client_apps.rst

```diff
@@ -3,12 +3,332 @@
 Quick Start Guide
```

Comment:
The content here should actually be in the Sample App Walkthrough and the Sample App Walkthrough content should be here.

Reply:
done